saleor/core/management/commands/cleardb.py

```diff
@@ -0,0 +1,108 @@
+staff accounts, service accounts, plugin configurations, site settings or navigation
+menus.
+"""
+
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+from ....account.models import User
+from ....checkout.models import Checkout
+from ....discount.models import Sale, Voucher
+from ....giftcard.models import GiftCard
+from ....order.models import Order
+from ....page.models import Page
+from ....payment.models import Payment, Transaction
+from ....product.models import Attribute, Category, Collection, Product, ProductType
+from ....shipping.models import ShippingMethod, ShippingZone
+from ....warehouse.models import Warehouse
+from ....webhook.models import Webhook
+
+
+class Command(BaseCommand):
+    help = "Removes data from the database preserving shop configuration."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--delete-staff",
+            action="store_true",
+            help="Delete staff user accounts (doesn't delete superuser accounts).",
+        )
+
+    def handle(self, **options):
+        if not settings.DEBUG:
+            raise CommandError("Cannot clear the database in DEBUG=True mode.")
+
+        Checkout.objects.all().delete()
+        self.stdout.write("Removed checkouts")
+
+        Transaction.objects.all().delete()
+        self.stdout.write("Removed transactions")
+
+        Payment.objects.all().delete()
+        self.stdout.write("Removed payments")
+
+        Order.objects.all().delete()
+        self.stdout.write("Removed orders")
+
+        Product.objects.all().delete()
+        self.stdout.write("Removed products")
+
+        ProductType.objects.all().delete()
+        self.stdout.write("Removed product types")
+
+        Attribute.objects.all().delete()
+        self.stdout.write("Removed attributes")
+
+        Category.objects.all().delete()
+        self.stdout.write("Removed categories")
+
+        Collection.objects.all().delete()
+        self.stdout.write("Removed collections")
+
+        Sale.objects.all().delete()
+        self.stdout.write("Removed sales")
+
+        ShippingMethod.objects.all().delete()
+        self.stdout.write("Removed shipping methods")
+
+        ShippingZone.objects.all().delete()
+        self.stdout.write("Removed shipping zones")
+
+        Voucher.objects.all().delete()
+        self.stdout.write("Removed vouchers")
+
+        GiftCard.objects.all().delete()
+        self.stdout.write("Removed gift cards")
+
+        self.stdout.write("Removed warehouses")
+        Warehouse.objects.all().delete()
+
+        Page.objects.all().delete()
+        self.stdout.write("Removed pages")
+
+        Webhook.objects.all().delete()
+        self.stdout.write("Removed webhooks")
+
+        # Delete all users except for staff members.
+        staff = User.objects.filter(Q(is_staff=True) | Q(is_superuser=True))
```

Comment:
The query `User.objects.filter(Q(is_staff=True) | Q(is_superuser=True))` will return the same results as `User.objects.filter(is_staff=True)`
```
In [20]: User.objects.filter(Q(is_staff=True) | Q(is_superuser=True))
Out[20]: <QuerySet [<User: michele.west@example.com>, <User: stacey.moore@example.com>, <User: alicia.clark@example.com>, <User: admin@example.com>, <User: james.smith@example.com>, <User: michelle.ross@example.com>]>
```

```
In [18]: User.objects.filter(Q(is_staff=True))
Out[18]: <QuerySet [<User: michele.west@example.com>, <User: stacey.moore@example.com>, <User: alicia.clark@example.com>, <User: admin@example.com>, <User: james.smith@example.com>, <User: michelle.ross@example.com>]>
```

Reply:
In the case of our example data, yes, but does `is_superuser=True` imply `is_staff=True`? These are two separate fields in the database.

Reply:
Yes, you are right. I thought that there are some correlations between these fields.